concepticondata/conceptlists/Anonby-2018-1500.tsv

```diff
@@ -0,0 +1,1501 @@
+Anonby-2018-1500-796	796			coming-of-age	بالغ شدن، به سن بلوغ رسیدن	bāleġ šodan, be senn-e boluġ residan	Human civilisation	Ceremonies	
+Anonby-2018-1500-797	797	203	MARRY	marry	ازدواج کردن، عروسی کردن	ezdeʋāj kardan, arusi kardan	Human civilisation	Ceremonies	Marriage
+Anonby-2018-1500-798	798	2432	MARRIAGE	marriage (state of wedlock)	ازدواج	ezdeʋāj	Human civilisation	Ceremonies	Marriage
+Anonby-2018-1500-799	799			engaged, betrothed	نامزد، نامزد شده	nāmzad, nāmzadšode	Human civilisation	Ceremonies	Marriage
+Anonby-2018-1500-800	800	3024	BRIDE PRICE	brideprice (for bride’s family)	شیربها	širbahā	Human civilisation	Ceremonies	Marriage
+Anonby-2018-1500-801	801	1259	WEDDING	wedding (ceremony)	 عروسی	arusi	Human civilisation	Ceremonies	Marriage
+Anonby-2018-1500-802	802	1266	BRIDE	bride	عروس	arus	Human civilisation	Ceremonies	Marriage
+Anonby-2018-1500-803	803	3282	GROOM	groom	داماد	dāmād	Human civilisation	Ceremonies	Marriage
+Anonby-2018-1500-804	804			polygamy	چندهمسری	čandhamsari	Human civilisation	Ceremonies	Marriage
+Anonby-2018-1500-805	805	902	ADULTERY	adultery	زنا	zenā	Human civilisation	Ceremonies	Marriage
+Anonby-2018-1500-806	806	3672	DIVORCE (SEPARATE)	divorce (v.)	طلاق گرفتن	talāġ gereftan	Human civilisation	Ceremonies	Marriage
+Anonby-2018-1500-807	807	3336	FUNERAL	funeral (at occasion of death)	(مراسم) خاکسپاری، دفن	(marāsem-e) xāksepāri, dafn	Human civilisation	Ceremonies	Funeral
+Anonby-2018-1500-808	808	3673	MOURNING	mourning	سوگواری، عزاداری	sugʋāri, azādāri	Human civilisation	Ceremonies	Funeral
+Anonby-2018-1500-809	809			wail, ululate (at funeral) (v.)	شیون کردن	šiʋan kardan	Human civilisation	Ceremonies	Funeral
+Anonby-2018-1500-810	810			console, comfort (v.)	تسلیت گفتن، تسلیت دادن	tasliyat goftan, tasliyat dādan	Human civilisation	Ceremonies	Funeral
+Anonby-2018-1500-811	811	767	CORPSE	corpse	جسد، جنازه	jasad,  jenāze	Human civilisation	Ceremonies	Funeral
+Anonby-2018-1500-812	812	1719	BURY	bury	دفن کردن، خاک کردن	dafn kardan, xāk kardan	Human civilisation	Ceremonies	Funeral
+Anonby-2018-1500-813	813	1773	GRAVE	grave	قبر	ġabr	Human civilisation	Ceremonies	Funeral
+Anonby-2018-1500-814	814	3284	GRAVEYARD	cemetery	قبرستان	ġabrestān	Human civilisation	Ceremonies	Funeral
+Anonby-2018-1500-815	815	619	ANIMAL	animal	حیوان	hayʋān	Animals		
+Anonby-2018-1500-816	816	3044	BOVINE	ox (general term), bovine	گاو	gāʋ	Animals	Domestic animals	Bovines
+Anonby-2018-1500-817	817	1008	BULL	bull	گاوِنر	gāʋ-e nar	Animals	Domestic animals	Bovines
+Anonby-2018-1500-818	818	1007	COW	cow (female)	گاوِماده	gāʋ-e māde	Animals	Domestic animals	Bovines
+Anonby-2018-1500-819	819			heifer	گوساله‌ی ماده	gusāle-ye māde	Animals	Domestic animals	Bovines
+Anonby-2018-1500-820	820			steer	گوساله‌ی پرواری	gusāle-ye parʋāri	Animals	Domestic animals	Bovines
+Anonby-2018-1500-821	821	1989	CALF	calf	گوساله	gusāle	Animals	Domestic animals	Bovines
+Anonby-2018-1500-822	822			herd (of cattle)	گله	galle	Animals	Domestic animals	Bovines
```

Comment:
Yes, this one could be mapped to CATTLE.

Reply:
I'd rather leave both of them unmapped. Cattle seems to me to refer to the general class of animals while a herd is the group in which these animals live or the way of livestock farming.

Reply:
I see that Snider-2004-1700-768 ("herd (cattle, sheep) (n)") is mapped to CATTLE and Snider-2004-1700-964 ("herd (of cattle)") is unmapped. This would need to be changed too. Dellert-2018-1016-133 ("Herde") is mapped to FLOCK (OF ANIMALS). Maybe we could map "herd (cattle, sheep) (n)" to this concept?

Reply:
In fact, 'herd' appears to be more general than 'cattle' and can apply to different animals, including wild ones. See, for example, here https://en.wikipedia.org/wiki/Herd:

> The term herd is generally applied to mammals, and most particularly to the grazing ungulates that classically display this behaviour. Different terms are used for similar groupings in other species; in the case of birds, for example, the word is flocking, but flock may also be used, in certain instances, for mammals, particularly sheep or goats. Large groups of carnivores are usually called packs, and in nature a herd is classically subject to predation from pack hunters.

This is a good reason to map the gloss to CATTLE, I think.